modules/services/window-managers/i3.nix

```diff
@@ -20,6 +20,15 @@ let
         description = "Whether to run command on each i3 restart.";
       };
 
+      notification = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Whether to enable startup-notification support for the command.
+          See --no-startup-id option description in the i3 user guide.
```

Comment:
Can use the [option](http://tdg.docbook.org/tdg/5.2/option.html) docbook element: `<option>--no-startup-id</option>`.

Reply:
This works well when I add it in the parameter description, but in the news, it just renders like `<option>--no-startup-id</option>`. Am I doing something wrong?

Reply:
Yeah, you'll have to keep it as `--no-startup-id` in the news entry. The news system is still very rudimentary and will not attempt to do any formatting of the message text. I.e., what you write there is shown as-is.

I'd like to make it support formatting at some point but haven't felt that it's very urgent 😉

Reply:
Maybe you could strip the formatting in the meantime.

Reply:
I think it'll be a bit tricky to reliably remove the formatting and still keep pretty plain text output. For now I think it's easier to just write plain text in the news entries and then when some markup is integrated we can just convert the news entries to the chosen markup.